icu4j/main/core/src/main/java/com/ibm/icu/text/RuleBasedNumberFormat.java

```diff
@@ -416,106 +401,91 @@
- *     <td style="width: 23;"></td>
- *     <td style="width: 165; vertical-align: top;">in fraction or default rule</td>
+ *     <td style="white-space: nowrap; vertical-align: top; padding-left: 1em; padding-right: 1em;">in fraction or default rule</td>
  *     <td>Isolate the number's integral part and format it.</td>
  *   </tr>
  *   <tr>
- *     <td style="width: 37;"></td>
- *     <td style="width: 23;"></td>
- *     <td style="width: 165; vertical-align: top;">in rule in fraction rule set</td>
+ *     <td style="white-space: nowrap; vertical-align: top; padding-left: 1em; padding-right: 1em;">in rule in fraction rule set</td>
  *     <td>Multiply the number by the rule's base value and format the result.</td>
  *   </tr>
- *   <tr>
- *     <td style="width: 37;"></td>
- *     <td style="width: 23;">==</td>
- *     <td style="width: 165; vertical-align: top;">in all rule sets</td>
+ *   <tr style="border-top: 1px solid black;">
+ *     <td style="white-space: nowrap;">==</td>
+ *     <td style="white-space: nowrap; vertical-align: top; padding-left: 1em; padding-right: 1em;">in all rule sets</td>
  *     <td>Format the number unchanged</td>
  *   </tr>
- *   <tr>
- *     <td style="width: 37;"></td>
- *     <td style="width: 23;">[]</td>
- *     <td style="width: 165; vertical-align: top;">in normal rule</td>
- *     <td>Omit the optional text if the number is an even multiple of the rule's divisor</td>
+ *   <tr style="border-top: 1px solid black;">
+ *     <td style="white-space: nowrap;" rowspan="6">[]</td>
+ *     <td style="white-space: nowrap; vertical-align: top; padding-left: 1em; padding-right: 1em;">in normal rule</td>
+ *     <td>Omit the optional text and rules if the number is an even multiple of the rule's divisor.
+ *     If the | symbol is present, the text and rules after the symbol and before the right bracket will be used instead of omission.</td>
  *   </tr>
  *   <tr>
- *     <td style="width: 37;"></td>
- *     <td style="width: 23;"></td>
- *     <td style="width: 165; vertical-align: top;">in negative-number rule</td>
+ *     <td style="white-space: nowrap; vertical-align: top; padding-left: 1em; padding-right: 1em;">in negative-number rule</td>
  *     <td>Not allowed.</td>
  *   </tr>
  *   <tr>
- *     <td style="width: 37;"></td>
- *     <td style="width: 23;"></td>
- *     <td style="width: 165; vertical-align: top;">in improper-fraction rule</td>
- *     <td>Omit the optional text if the number is between 0 and 1 (same as specifying both an
- *     x.x rule and a 0.x rule)</td>
+ *     <td style="white-space: nowrap; vertical-align: top; padding-left: 1em; padding-right: 1em;">in improper-fraction rule</td>
+ *     <td>Omit the optional text and rules if the number is between 0 and 1 (same as specifying both an
+ *     x.x rule and a 0.x rule).
+ *     If the | symbol is present, the text and rules after the symbol and before the right bracket will be used instead of omission.</td>
```

Comment:
This sentence (in all the places it appears) sounds a little clunky to me.  Maybe something like "Omit the optional text and rules if (blah blah blah).  If the | symbol is present, use the text before the | symbol if (blah blah blah) and use the text after the | symbol if (NOT blah blah blah)".  It's longer and more work (you can't just copy-paste), but it'd be clearer and read a little better.

Reply:
Sure I’ll contemplate better wording.

Reply:
Please check the new wording.  I had to shuffle the order around and use bullet points so that it's easier to read.